src/elements/form-field/form-field/form-field.ts

```diff
@@ -399,7 +401,9 @@ class SbbFormFieldElement extends SbbNegativeMixin(SbbHydrationMixin(LitElement)
   }
 
   private _isInputEmpty(): boolean {
-    if (this._input instanceof HTMLInputElement) {
+    if (this.querySelector('sbb-chip-group')) {
+      return this._isInputValueEmpty() && this.querySelector('sbb-chip-group')!.value.length === 0;
```

Comment:
with my suggestion, the "!" should be obsolete.
```suggestion
      return this._isInputValueEmpty() && chipGroup.value.length === 0;
```